Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -149,6 +149,14 @@ class ActionController extends AbstractController
      */
     protected $logger;
 
+    /**
+     * Feature flag to enable the potentially breaking support of validation for dynamic types specified with `__type` argument or in the `PropertyMapperConfiguration`.
+     * Note: This will be enabled by default in a future version.
+     * See https://github.com/neos/flow-development-collection/pull/1906
```

Comment:
IMO we should link to the _issue_ instead (https://github.com/neos/flow-development-collection/issues/1905) since there might be multiple PRs attached to it (at some point). Also we can collect more generic infos in the issue